hoptimator-cli/src/main/java/com/linkedin/hoptimator/HoptimatorCliApp.java

```diff
@@ -55,6 +55,38 @@ protected int run(String[] args) throws IOException {
     sqlline.updateCommandHandlers(commandHandlers);
     return sqlline.begin(args, null, true).ordinal();
   }
+
+  private static InsertInto parseInsertInto(String s) {
+      String sql = s.trim();
+      if (!sql.startsWith("insert into ")) {
```

Comment:
Any reason for not having case insensitive support for `insert into` but we have case insensitive support for `SELECT`?

Reply:
Good idea! I just cargo-culted this behavior from sqlline, but I don't see a reason to not make it case insensitive.